docs/docusaurus/versioned_docs/version-0.17.23/components/_prerequisites.jsx

```diff
@@ -73,10 +74,10 @@ export default class Prerequisites extends React.Component {
     } else if (this.props.requireDatasource === 'SQL') {
       returnItems.push(<li><a href='/guides/connecting_to_your_data/datasource_configuration/how_to_configure_a_sql_datasource'>A Datasource configured to access your source data.</a></li>)
     } else if (this.props.requireDatasource === true) {
-      returnItems.push(<li><a href='/guides/connecting_to_your_data/connect_to_data_overview'>A Datasource configured to access your source data</a></li>)
+      returnItems.push(<li><VersionedLink to='/guides/connecting_to_your_data/connect_to_data_lp'>A Datasource configured to access your source data</VersionedLink></li>)
```

Comment:
Actually commenting about lines 66-76 above.  Do those need to be updated to use the versioned link as well?  I believe they were originally the 0.15.x links that were removed by my PR, and that may have resulted in the format not being the same as whatever you were searching for to find links that needed update to use the VersionedLink.